chains/rootstock/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://rootstock.blockscout.com/api
+    family: blockscout
+    name: Blockscout
+    url: https://rootstock.blockscout.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 30
+  reorgPeriod: 4
+chainId: 30
+deployer:
+  # Deployment in discussion between Rootstock <> Abacus Works.
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Rootstock
+domainId: 30
+isTestnet: false
+name: rootstock
+nativeToken:
+  decimals: 18
+  name: Rootstock Smart Bitcoin
+  symbol: RBTC
+protocol: ethereum
+rpcUrls:
+  # Note: Using Rootstock RPC api here which contains APIKEY in url.
+  # Recommended way is to get a new APIKEY whoever wants to run Hyperlane on Rootstock.
+  # RPC API Dashboard: https://dashboard.rpc.rootstock.io
+  - http: https://rpc.mainnet.rootstock.io/kXhXHf6TnnfW1POvr4UT0YUvujmuju-M
+transactionOverrides:
+  gasLimit: 6800000
```

Comment:
a link to dev docs would be great if possible!

Reply:
Fixed!